transform_assumption.ml

```diff
@@ -210,23 +210,23 @@ let hoist_assumption : transform_instructions = fun ({instrs; _} as inp) ->
   let changed = ref false in
   let push_osr pc =
     match[@warning "-4"] instrs.(pc) with
-    | Osr {label; cond; target; map} ->
+    | Osr {label; cond; target; var_map; frame_maps} ->
       let try_push c =
         let cond_vars = expr_vars c in
         begin match find_candidate_osr c cond_vars (pc-1) None with
         | None -> true
         | Some pc' ->
           changed := true;
           begin match[@warning "-4"] instrs.(pc') with
-          | Osr {label; cond; target; map} ->
-            instrs.(pc') <- Osr {label; cond = c::cond; target; map}
+          | Osr {label; cond; target; var_map; frame_maps} ->
+              instrs.(pc') <- Osr {label; cond = c::cond; target; var_map; frame_maps}
```

Comment:
thanks, somehow merlin update changed identation after `->` from 2 to 4 spaces :/